/**:
  ros__parameters:
    imu_topic: "/tartanair/imu"
    laser_topic: "/tartanair/velodyne"
    sensor: "velodyne"
    use_imu_roll_pitch: false
    
    world_frame: "map"
    world_frame_rot: "map_rot"
    sensor_frame: "sensor"
    sensor_frame_rot: "sensor_rot"

    imu_acc_x_limit: 3.0
    imu_acc_y_limit: 3.0
    imu_acc_z_limit: 5.0

    # Feature Extraction Params
    feature_extraction_node:
        scan_line: 16 
        min_range: 0.75    #<!-- remove too closed points -->
        filter_point_size: 5

    # Laser Mapping Params
    laser_mapping_node:
        mapping_line_resolution: 0.1
        mapping_plane_resolution: 0.2
        max_iterations: 30
        max_surface_features: 2000

        # For localization mode
        localization_mode: false
        read_pose_file: false
        init_x: 0.0
        init_y: 0.0
        init_z: 0.0
        init_roll: 0.0
        init_pitch: 0.0
        init_yaw: 0.0 

    # IMU Preintegration Params
    imu_preintegration_node:
        lidar_correction_noise: 0.02

        acc_n: 0.3         # accelerometer measurement noise standard deviation. #0.2
        gyr_n: 0.07         # gyroscope measurement noise standard deviation.     #0.05
        acc_w: 0.005        # accelerometer bias random work noise standard deviation.  #0.02
        gyr_w: 1.5e-05       # gyroscope bias random work noise standard deviation.     #4.0e-5
        g_norm: 9.80511    # 0 means you need to provide both imu-camera and laser-camera extrinsic

